kits/rosie/rosie_demo.cpp

```diff
@@ -424,16 +465,16 @@ int main(){
       } else {
         t = omni_base_traj->getDuration();
       }
-      Eigen::VectorXd chassis_cmd_vel(base.num_wheels_);
-      Eigen::VectorXd chassis_cmd_acc(base.num_wheels_);
-      Eigen::VectorXd chassis_cmd_jerk(base.num_wheels_);
+      Eigen::VectorXd chassis_cmd_vel(base_num_wheels);
+      Eigen::VectorXd chassis_cmd_acc(base_num_wheels);
+      Eigen::VectorXd chassis_cmd_jerk(base_num_wheels);
```

Comment:
This is very frustrating. This isn't your fault, and you're doing everything right with the current API... However, a ton of these variables in these examples have a statically set size, which will never change. Therefore, the trajectory and kinematics APIs **should** support passing in compile time sized vectors, which helps not only generate better code but also makes it harder to accidentally pass in an incorrectly sized matrix.

TL;DR: The API lacks proper functionality to allow something like `Eigen::Matrix<double, BaseNumWheels, 1> chassis_cmd_vel`, etc